src/lib/pages/backoffice/Patron/PatronDetails/ItemsSearch/ItemsSearch.js

```diff
@@ -82,42 +72,19 @@ export default class ItemsSearch extends Component {
         results={results}
         isLoading={isLoadingSearch}
         executedSearch={executedSearch}
-        clearSearchQuery={this.clearSearchQuery}
       />
     );
   };
 
-  clearSearchQuery = () => {
-    const { clearResults } = this.props;
-
-    // eslint-disable-next-line react/no-unused-state
-    this.setState({ prevSearchQuery: '' });
-    clearResults();
-  };
-
   render() {
-    const {
-      items,
-      isLoading,
-      error,
-      queryString,
-      updateQueryString,
-    } = this.props;
+    const { items, isLoading, error, queryString } = this.props;
     return (
       <>
         <Container className="spaced">
-          <SearchBar
-            action={{
-              icon: 'search',
-              onClick: this.onSearchClickHandler,
-            }}
-            currentQueryString={queryString}
-            updateQueryOnChange
-            executeSearch={this.executeSearch}
-            onKeyPressHandler={this.onKeyPressHandler}
-            updateQueryString={updateQueryString}
+          <SearchBarILS
+            onSearchHandler={this.executeSearch}
             placeholder="Type or paste to search for physical copies..."
-            onPaste={e => this.onPasteHandler(e)}
+            onPasteHandler={this.onPasteHandler}
```

Comment:
same for the `ENTER` button here, or maybe it works differently now?

Reply:
This is the same case, but we override the paste behaviour